### ISSUE TITLE
[Question] Will you continue working in the documentation of this project?

### ISSUE THREAD

**Issue by dave-zz** [johnazariah/orleans-contrib-universalsilo#54]
Good afternoon,

My name is David and I just watched [an old conference video about Orleans](https://www.youtube.com/watch?v=7OVU9Mqqzgs). I already found the sample code you presented, and I got here after taking a look at your profile.

Originally I was going to use Akka.net, but I'm the only developer for this project and I don't have so much time to understand the Akka implementation of the actor model. Orleans is convenient for me because I find its interface-based implementation easier to understand.

I'm very interested in this project because it's exactly what I want to do. I noticed the documentation is not complete and decided to ask you about it since it would be of great help.

Thanks in advance!

**Comment by johnazariah** [comment]
Yes, I am working on it - sorry I've been a bit pre-occupied over the last few months. But I'm back on this now...
